mlocal/frags/Makefile.stub

```diff
@@ -133,7 +133,7 @@ $(libruntime): $(libutil_OBJ)
 	$(V)$(AR) rcs $@ $(libutil_OBJ)
 
 # starter
-$(starter): $(go_OBJ) $(starter_OBJ) $(libruntime)
+$(starter): $(go_OBJ) $(starter_SOURCE) $(starter_OBJ) $(libruntime)
```

Comment:
If the idea here is to force `go build ...` to do a fresh build when the `starter.c` file has been changed, then this change won't work. The problem is that `go build` does not understand when a C file has changed, so the cache does not become invalidated. We'll still run the `go build` command, but it won't actually do a fresh compile with the new C code.

I think a fix for this should be submitted in a separate PR, and it can be addressed in a different way. Something like this:

```
starter_SOURCE := ...
starter_DUMMY := $(BUILDDIR)/src/runtime/starter/c/starter.c

$(starter_DUMMY): $(starter_SOURCE)
    go clean -cache
    touch $@

$(starter): $(go_OBJ) $(starter_DUMMY) $(starter_OBJ) $(libruntime)
    ...
```

Basically, create an empty target file in `BUILDDIR` which has the `starter.c` file as a prerequisite. If `starter.c` changes, we do `go clean -cache` (to force go to fully rebuild from scratch) and then touch the `starter_DUMMY` file to update it. Since `$(starter)` has that file as a prerequisite, if we ever change `starter.c`, the go build cache will get cleaned before we compile the `starter` binary.

Reply:
The idea is to recompile `starter` when `starter.c` is modified nothing more. Why not just simply disable go cache for the starter part by running `GOCACHE=off go build` ? That could limit bad surprises

Reply:
Or `go build -a` ?

Reply:
Yea that would work as well. But that also means if any of `$(go_OBJ) $(starter_OBJ) $(libruntime)` are changed we always recompile the entire thing. Basically this means we lose caching at all, which isn't ideal. The idea is that **if and only if** `starter.c` is modified, we run without cache

Reply:
Ok @bauerm97 changes applied